### DISCUSSION TITLE
Any badges?

### DISCUSSION THREAD

**Discussion by serkor1** [posit-dev/positron#5190] [Q&A]
Hey guys,

Is there any badges (or somehting of the sort) we can pin on our repositories so we can show support and love for Positron? :thinking: 
(This is would my way of contributing to the project, as I can't program in w/e language this is being developed in)

Best,

**Comment by lionel-** [comment]
Not that I know of but you could link to https://positron.posit.co/?

**Reply by serkor1** [reply]
Not a bad suggestion! I'll start with that I guess 🤷‍♀️

**Reply by lionel-** [reply]
Thanks for your support!
